lib/client-api/src/StoryStoreFacade.ts

```diff
@@ -28,6 +28,7 @@ export interface GetStorybookKind<TFramework extends AnyFramework> {
   stories: GetStorybookStory<TFramework>[];
 }
 
+// TODO: talk to michael about the removal of this
```

Comment:
@shilman let's chat about this, how to remove.

Reply:
This is new @ndelangen, so not a candidate for removal.

Reply:
We should not be using `deprecate` to give this warning to users then.